### PR TITLE
fix comments and strings in language_cmake.lua

### PR DESCRIPTION
**Pull request by ndytts**
Changed comment from "//" to "#" in language_cmake, added block_comment, and others small changes

### CODE REVIEW

**Comment by Guldoman** [comment]
Would you mind also bumping the plugin version in the manifest?

File: plugins/language_cmake.lua [hunk 1]
```diff
@@ -4,14 +4,17 @@ local syntax = require "core.syntax"
...
+  block_comment = { "#[[", "]]" },
   patterns = {
-    { pattern = { "#", "[^\\]\n" },     type = "comment"  },
-    { pattern = { '"', '"', '\\' },     type = "string"   },
-    { pattern = { "'", "'", '\\' },     type = "string"   },
-    { pattern = "[%a_][%w_]*%f[(]",     type = "function" },
-    { pattern = "[%a_][%w_]*",          type = "normal"   },
-	{ pattern = "%${[%a_][%w_]*%}",     type = "operator"   },
+    { pattern = { "#%[%[", "%]%]"},      type = "comment"  },
+    { pattern = { '#%[=*%[', '%]=*%]' }, type = "comment"  },
```
Comment (Guldoman) [inline plugins/language_cmake.lua:11]: I think the first one isn't needed, as the second one covers it.

Comment (ndytts) [reply plugins/language_cmake.lua:11]: true, the * covers from 0 to anyone, I forgot...

**Comment by Guldoman** [comment]
Looks good, thanks!

**Pull request merged**